lib/wpc_proto/wpc_proto.c

```diff
@@ -52,23 +52,36 @@ app_proto_res_e WPC_Proto_initialize(const char * port_name,
                                      char * gateway_id,
                                      char * gateway_model,
                                      char * gateway_version,
-                                     char * sink_id)
+                                     char * sink_id,
+                                     unsigned int max_poll_fail_duration_s,
```

Comment:
Not related to this change.

And do we really want to add it as parameter instead of keeping it internal.

Same for max_fragment_duration_s and especially if we want to move later to a version with a different algo (keep it until full).
So for now I wouldn't add too much parameter to proto_api that will probably be removed later.